open_discussions/views.py

```diff
@@ -30,19 +31,25 @@ def index(request):
         auth_url = payload.get("auth_url", None)
         session_url = payload.get("session_url", None)
         username = payload.get("username", None)
+        site_key = payload.get("key", settings.OPEN_DISCUSSIONS_DEFAULT_SITE_KEY)
     except jwt.InvalidTokenError:
         auth_url = None
         session_url = None
         username = None
+        site_key = settings.OPEN_DISCUSSIONS_DEFAULT_SITE_KEY
+
+    site = Site.objects.filter(key=site_key).first()
 
     js_settings = {
         "gaTrackingID": settings.GA_TRACKING_ID,
         "public_path": public_path(request),
         "auth_url": auth_url,
-        "session_url": session_url,
+        "session_url": session_url if site is None else site.session_url,
         "max_comment_depth": settings.OPEN_DISCUSSIONS_MAX_COMMENT_DEPTH,
-        "micromasters_external_login_url": settings.MICROMASTERS_EXTERNAL_LOGIN_URL,
-        "micromasters_base_url": settings.MICROMASTERS_BASE_URL,
+        "micromasters_external_login_url":
+            settings.MICROMASTERS_EXTERNAL_LOGIN_URL if site is None else site.login_url,
```

Comment:
Would these settings essentially be deprecated after this PR?

Reply:
Yeah, actually this slipped through the cracks. I meant to not update these as I don't want complicated code here. And the record for MM won't exist until we create it in django-admin so I want to phase in these changes as outlined in #444. I'll revert this bit of code.